### ISSUE TITLE
after login, redirect to previous page

### ISSUE THREAD

**Issue by logust79** [phenopolis/phenopolis#78]

**Comment by IsmailM** [comment]
Can you explain further? are you logging in as demo or another account?

At the moment, if the login fails, it redirects to the home page (I'm fixing later today/tomorrow) - is that what you see...

**Comment by logust79** [comment]
Say my first intention today is to visit /gene/ABCA4 (perhaps to follow a previous day's clue), it would ask me to login. But after login, it redirects me to homepage. The preference would be to redirect to /gene/ABCA4

**Comment by pontikos** [comment]
So at the moment the login page redirects you to the search page but I
think what @logust79 is suggesting is that login redirects you to the
referrer ( in this case /gene/ABCA4 ).

**Comment by IsmailM** [comment]
yep, was this implemented in the older version of the login page?

Easiest way would be to use referral URL in the header...

I'll take a look at implementing it later today/tomorrow

**Comment by logust79** [comment]
Great Thank you!

**Comment by pontikos** [comment]
Thanks @IsmailM , I don't think it was to be honest but it would be a nice feature.
